Lantiq Image: Scan profiles for settings and create full list
- get SUBPROFILES and specific DTS from profile definiton

diff --git a/target/linux/lantiq/image/Makefile b/target/linux/lantiq/image/Makefile
--- a/target/linux/lantiq/image/Makefile
+++ b/target/linux/lantiq/image/Makefile
@@ -9,6 +9,26 @@
 #EASY50712 ARV3527P
 
 include $(TOPDIR)/rules.mk
+
+ifndef Profile
+define Profile
+  SUBPROFILES:=
+  DTS:=
+  $(eval $(call Profile/$(1)))
+  ALL_PROFILES += $(1)
+  ifneq ($(DTS),)
+    DTS_$(1):=$(DTS)
+  endif
+  ifneq ($(SUBPROFILES),)
+    SUBPROFILES_$(1):=$(SUBPROFILES)
+  endif
+
+  ifeq ($(CONFIG_TARGET_$(call target_conf,$(BOARD)_$(if $(SUBTARGET),$(SUBTARGET)_))$(1)),y)
+    PROFILE=$(1)
+  endif
+endef
+endif
+
 include $(INCLUDE_DIR)/image.mk
 include $(INCLUDE_DIR)/version.mk
 
